.gitignore

```diff
@@ -55,3 +55,6 @@ built_pkgs
 __pycache__
 .circleci/config.yml-local
 .env
+compile_commands.json
```

Comment:
```suggestion
compile_commands.json
compile_commands.sqlite3
```

or maybe `compile_commands.*`? (`bear` creates the sqlite database during builds, but apparently cleans it up afterwards)

Reply:
Done. Replaced it with `compile_commands.*`.